RecoBTag/SoftLepton/src/MuonTagger.cc

```diff
@@ -35,8 +36,8 @@ float MuonTagger::discriminator(const TagInfoHelper& tagInfo) const {
     bool flip(false);
     if(m_selector.isNegative()) {
       int seed=1+round(10000.*properties.deltaR);
```

Comment:
a random (that is not so random) in reco ???
Could somebody explain the rationale of all this?